examples/mlperf/helpers.py

```diff
@@ -225,3 +225,14 @@ def get_data_bert(GPUS:list[str], it):
   data: dict[str, Tensor] = next(it)
   for key in data.keys(): data[key].shard_(GPUS, axis=0)
   return data
+
+def get_fake_data_bert(GPUS:list[str], BS:int):
+  return {
+    "input_ids": Tensor.zeros((BS, 512), dtype=dtypes.float32).shard_(GPUS, axis=0),
```

Comment:
add `contiguous` to these `zeros` to make these real buffers. for now we call `contiguous` when sharding but this might change in the future